matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/timeline/TimelineEvent.kt

```diff
@@ -184,7 +184,7 @@ fun TimelineEvent.isRootThread(): Boolean {
 fun TimelineEvent.getTextEditableContent(formatted: Boolean): String {
```

Comment:
I struggle to understand the purpose of the `formatted` parameter.
Is it a necessary parameter, could we write?
`val lastContentBody = (lastMessageContent as? MessageContentWithFormattedBody)?.formattedBody ?: lastMessageContent?.body.orEmpty()`

If necessary, could you add some KDoc about it?

Reply:
Okay I see. Thanks for explaining!